chaco/tests/test_image_plot.py

```diff
@@ -152,3 +165,31 @@ def test_vertical_bottom_right(self):
         # that is flipped vertically and horizontally.
         self.verify_result_image(RGB, (IMAGE.T)[::-1, ::-1],
                                  origin='bottom right', orientation='v')
+
+    # regression test for enthought/chaco#528
+    def test_resize_to_zero(self):
+        class TestResize(HasTraits):
+            plot = Instance(Component)
+
+            # at the exact size of (101, 101) for the component editor the
+            # image is 0x0 on the screen and the failure was triggered
+            traits_view = View(
+                Group(
+                    UItem('plot', editor=ComponentEditor(size=(101, 101))),
+                ),
+                resizable=True,
+            )
+
+            def _plot_default(self):
+                pd = ArrayPlotData(image=face())
+                plot = Plot(pd)
+                plot.img_plot('image')
+
+                return plot
+        
+        test_obj = TestResize()
+        tester = UITester()
+
+         # should not fail
+        with tester.create_ui(test_obj) as ui:
+            pass
```

Comment:
this test is a little ugly namely from the 101 magic number.  I was unsure how to better write this test.

Without the fix this test fails, with it it passes.